src/views/Peers.tsx

```diff
@@ -154,99 +181,141 @@ export const Peers = () => {
+                                        <Radio.Group
+                                            options={optionsOnOff}
+                                            onChange={onChangeOnOff}
+                                            value={optionOnOff}
+                                            optionType="button"
+                                            buttonStyle="solid"
+                                        />
+                                        <Select value={pageSize.toString()} options={pageSizeOptions} onChange={onChangePageSize} className="select-rows-per-page-en"/>
+                                    </Space>
+                                </Col>
+                                <Col xs={24}
+                                     sm={24}
+                                     md={5}
+                                     lg={5}
+                                     xl={5}
+                                     xxl={5} span={5}>
+                                    <Row justify="end">
+                                        <Col>
+                                            <Link to="/add-peer" className="ant-btn ant-btn-primary ant-btn-block">Add Peer</Link>
+                                        </Col>
+                                    </Row>
+                                </Col>
+                            </Row>
+                            {failed &&
+                                <Alert message={failed.code} description={failed.message} type="error" showIcon closable/>
+                            }
+                            {loading && <Loading/>}
+                            <Card bodyStyle={{padding: 0}}>
+                                <Table
+                                    pagination={{pageSize, showSizeChanger: false, showTotal: ((total, range) => `Showing ${range[0]} to ${range[1]} of ${total} peers`)}}
+                                    className="card-table"
+                                    showSorterTooltip={false}
+                                    scroll={{x: true}}
+                                    dataSource={dataTable}>
+                                    <Column title="Name" dataIndex="Name"
+                                            onFilter={(value: string | number | boolean, record) => (record as any).Name.includes(value)}
+                                            sorter={(a, b) => ((a as any).Name.localeCompare((b as any).Name))} />
+                                    <Column title="IP" dataIndex="IP"
+                                            sorter={(a, b) => {
+                                                const _a = (a as any).IP.split('.')
+                                                const _b = (b as any).IP.split('.')
+                                                const a_s = _a.map((i:any) => i.padStart(3, '0')).join()
+                                                const b_s = _b.map((i:any) => i.padStart(3, '0')).join()
```

Comment:
Codacy has a fix for the issue: [Replace `············································const·b_s·=·_b.map((i:any)·=>·i.padStart(3,·'0')).join()` with `····················const·b_s·=·_b.map((i:·any)·=>·i.padStart(3,·"0")).join();`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
                    const b_s = _b.map((i: any) => i.padStart(3, "0")).join();
```